scripts/gcloud_adapter.py

```diff
@@ -65,6 +159,18 @@ def get_currently_served_version(app_name):
     return listed_versions[:listed_versions.index(' ')]
 
 
+def switch_version(app_name, release_version):
+    """Switches to the release version and migrates traffic to it.
+
+    Args:
+        app_name: str. The name of the GCloud project.
+        release_version: str. The release version to switch to.
```

Comment:
Call this version_to_switch_to (this function should not know about its context).

Reply:
Done.